code/modules/mob/living/carbon/carbon.dm

```diff
@@ -516,7 +516,6 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	return
 
 /mob/living/carbon/throw_item(atom/target)
-	throw_mode_off()
 	if(!target || !isturf(loc))
 		return
```

Comment:
If you don't have a `throw_mode_off` before every return, some things that used to turn off throw mode won't anymore.